cmd/rekor-cli/app/root.go

```diff
@@ -16,16 +16,24 @@
 package app
 
 import (
-	"errors"
 	"fmt"
-	"net/url"
-	"os"
 	"strings"
 
 	homedir "github.com/mitchellh/go-homedir"
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
 	"github.com/spf13/viper"
+
+	"github.com/sigstore/rekor/pkg/log"
+
+	// these imports are to call the packages' init methods
```

Comment:
This feels strange to me, but maybe I'm not familiar enough with the flags/command library in use here? If this has to know about its sub commands -- should it not just be responsible for initializing them?

Reply:
before this change, the types weren't imported because we were directly creating objects of each type. Since the object creation code now lies in `pkg/types/...`, we need to trigger the type's `init()` methods to fire to populate the type & version maps accordingly, so that's why we have the imports here.

Reply:
yeah I think my problem is that we need to trigger the `init()` functions explicitly like without ever actually using anything from the package explicitly. The pattern just feels off to me, but I dunno, I don't have any better/alternative solutions.